scripts/publish.sh

```diff
@@ -251,7 +286,7 @@ check_parameter() {
 	fi
 }
 
-sync_latest_image() {
+sync_image_version() {
```

Comment:
Will there ever be a case where the AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB is an empty string "" because we:

1. Merged the code
2. Have not finished the release pipeline yet
3. Sync tasks begin

Would there be some error in this case where the sync tasks fail?

Reply:
Its still first set to the top version from the python script sorted list. And then it can be set to the version in the AWS_FOR_FLUENT_BIT version file instead if we are not publishing a latest release. So I think it can't be empty string. Do you see a case?

Reply:
I actually don't see any way that AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB can be an empty string any more. It will be latest if not the AWS_FOR_FLUENT_BIT_VERSION as you describe in your comment